eessi-2023.06-eb-4.8.2-2022a.yml

```
@@ -35,3 +35,8 @@ easyconfigs:
   - WSClean-3.4-foss-2022a:
       options:
         from-pr: 19119
+  - Pillow-9.1.1-GCCcore-11.3.0.eb:
+      # avoid that hardcoded paths like /usr/include are used in build commands
+      options:
+        from-pr: 19266
```

@TopRichard looks like you made a typo in the PR number, this should be:
```suggestion
        from-pr: 19226
```